src/ui.js

```diff
@@ -4350,6 +4362,25 @@ class Panel {
                         return output;
                     });
                 })
+        ).add(
+          // The embed button.
+          new DOM.Element("button").add("Get embed code")
+              .listen("click", () => {
+                  display_export_pane("base64", (output) => {
+                      const URL_prefix = window.location.href
+                          .replace(/(\/index\.html)?\?.*$/, "")
+                      return {
+                          data: `<!-- ${URL_prefix} -->
+<div class="quiver-embed">
+    <iframe src="${output.data}&macro_url=${
```

Comment:
This needs to take into account the case where `ui.macro_url` is `null`, in which case it ought not  to be included in the query string. (It might be worth abstracting this logic so "Get shareable link" and "Get embed code" share the same method.)